pkg/kev/kev_reconcile_test.go

```diff
@@ -457,6 +457,42 @@ var _ = Describe("Reconcile", func() {
 				Expect(mErr).NotTo(HaveOccurred())
 			})
 		})
+
+		Context("when healthcheck is overridden by overlay", func() {
+			Context("liveness tcp", func() {
+				BeforeEach(func() {
+					workingDir = "testdata/reconcile-healthcheck-tcp"
+					overrideFiles = []string{workingDir + "/docker-compose.kev.dev.yaml"}
+				})
+
+				It("should have a valid tcp", func() {
+					labels := env.GetServices()[0].GetLabels()
+
+					Expect(labels).Should(
```

Comment:
Small nitpick. With historically use `Expect` with `.To` / `.NotTo` instead of `.Should` / `.ShouldNot`. They are functionally identical but would make sense to stick to same style.